contracts/ConsensusUtils.sol

```diff
@@ -20,6 +20,7 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
   uint256 public constant CYCLE_DURATION_BLOCKS = 34560; // 48 hours [48*60*60/5]
   uint256 public constant SNAPSHOTS_PER_CYCLE = 0; // snapshot each 288 minutes [34560/10/60*5]
   uint256 public constant DEFAULT_VALIDATOR_FEE = 15e16; // 15%
+  uint256 public constant VALIDATOR_PRODUCTIVITY = 7; //must be a whole number (1-10) 1 = 10%, 2 = 20%..... 10=100%
```

Comment:
So to be jailed validator needs to to mine less then 70% of his blocks. right?
We use other convention for percentages in the script. see for example:
`  uint256 public constant DEFAULT_VALIDATOR_FEE = 15e16; // 15%`
Can you do the same? Want to keep consistency